rotkehlchen/tests/api/test_assets.py

```diff
@@ -642,6 +658,8 @@ def test_search_assets_with_levenshtein(rotkehlchen_api_server):
     result = assert_proper_response_with_result(response)
     assert len(result) <= 50
     assert_substring_in_search_result(result, 'ETH')
+    # check that Ethereum(ETH) appear at the top of result.
```

Comment:
I want to see exchaustive tests as I said. I need you to have a test that checks that the expected behavior is seen with `treat_eth2_as_eth` as True and `treat_eth2_as_eth` as False